src/getHumanAge.test.js

```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be an object', () => {
+    expect(typeof getHumanAge(15, 16))
+      .toEqual('object');
+  });
```

Comment:
```suggestion
  test('should return an array', () => {
    expect(getHumanAge(15, 16))
      .toBeInstanceOf(Array);
  });
```
because it's an array